webapp/channels/src/components/drafts/placeholder_scheduled_post_title/placeholder_scheduled_posts_title.tsx

```diff
@@ -17,6 +19,13 @@ export default function PlaceholderScheduledPostsTitle({type}: Props) {
         />
     );
 
+    const tooltipText = (
+        <FormattedMessage
+            id='scheduled_posts.row_title_thread.placeholder_tooltip'
+            defaultMessage={'This was scheduled in a private channel you are no member of.'}
```

Comment:
So the `No destination`  can appear even when a channel is hard deleted somehow. So, lets use a generic error message - `The channel either doesn't exist or you do not have access to it`.